plugins/modules/open_iscsi.py

```diff
@@ -403,53 +405,54 @@ def main():
     if show_nodes:
         result['nodes'] = nodes
 
-    if login is not None and not check_rc:
-        result['devicenodes'] = []
-        for index_target in nodes:
-            loggedon = target_loggedon(module, index_target, portal, port)
+    if login is not None:
+        # If check_rc=False we will try to login to all available target nodes
```

Comment:
Given that the semantics of that variable mean "login to all available nodes or not" maybe its name should not be `check_rc`. The more meaningful the coding elements are, the less one needs to explain them. Just my $0.02

Reply:
I added a variable `login_to_all_nodes` for If-statements, thanks!